library/std/src/sys/unix/process/process_common.rs

```diff
@@ -223,7 +223,7 @@ impl Command {
     pub fn get_groups(&self) -> Option<&[gid_t]> {
         self.groups.as_deref()
     }
-
+    #[allow(dead_code)]
```

Comment:
It's probably better to not put this attribute here and instead put something like `let _ = self.get_closures();` with a comment in the vxworks code. Then it's more explicit that the pre_exec closures are ignored on vxworks.

(Ideally we shouldn't even expose `pre_exec` on platforms that don't have that ability, but it's part of the `unix` extension trait for `Command`.)

Reply:
Added your suggestion with a comment.